src/components/SimpleAppBar.tsx

```diff
@@ -5,6 +5,7 @@ import { UploadControll } from './UploadControll'
 
 interface SimpleAppBarProps {
   uploadsetdata: () => void
+  notificationControll: (onoff: boolean) => void
```

Comment:
```suggestion
  notificationControl: (onoff: boolean) => void
```

Reply:
also I would avoid naming that argument onoff. Use something like `isOpen`.

Reply:
done